client/clientimpl_test.go

```diff
@@ -1501,6 +1525,8 @@ func createPackageTestCase(name string, downloadSrv *httptest.Server) packageTes
 		expectedSignature: map[string][]byte{
 			"package1": {6, 7},
 		},
+
+		temporaryStatuses: make([]*protobufs.PackageStatuses, 0),
```

Comment:
Nit: I think a `nil` slice should also work here, no need to allocate zero-length slice.